src/components/Sidebar/Sidebar.css

```diff
@@ -0,0 +1,25 @@
+.Sidebar-container {
+  background-color: #000000;
+  position: fixed;
+  width: 220px;
+  height: 100vh;
+  margin-top: 1px;
+}
+.sidebar-links {
+  padding: 16px 20px;
+  font: 16px Roboto;
+  text-decoration: none;
+  color: white;
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+}
+.sidebar-icons {
+  padding-right: 20px;
+  height: 30px;
+  width: 20px;
+}
+
+.sidebar-links:hover {
+  background-color: #536075;
```

Comment:
Same color variables here.
You're using #000000 for black, so you can just make a variable --COLOR-black